aiidalab_qe/app/steps.py

```diff
@@ -1077,6 +1166,11 @@ def _extract_report_parameters(
         ] = builder.bands.bands_kpoints_distance.value
         parameters["nscf_kpoints_distance"] = builder.pdos.nscf.kpoints_distance.value
 
+        if builder.relax.base["pw"]["parameters"]["SYSTEM"]["tot_charge"]:
```

Comment:
```suggestion
        tot_charge = builder.relax.base["pw"]["parameters"]["SYSTEM"]["tot_charge"]
        if total_charge:
```
This better move to above under the comment where said why the value getting from `builder.relax.base` is feasible.  

I also don't quite understand, the `tot_charge` can be a `None`?

Reply:
Yeah it can be done, it they do not apply advanced settings